test/regress/regress1/sygus/interpol_cosa_1.smt2

```diff
@@ -1,9 +1,9 @@
-; COMMAND-LINE: --produce-interpols=conjecture --sygus-active-gen=enum --check-interpols
+; COMMAND-LINE: --produce-interpols=conjecture --sygus-enum=fast --check-interpols
 ; SCRUBBER: grep -v -E '(\(define-fun)'
 ; EXIT: 0
 (set-logic ALL)
 (set-option :produce-interpols conjecture)
-(set-option :sygus-active-gen enum)
+(set-option :sygus-enum fast)
```

Comment:
Is this needed?

Reply:
Yes, this update fixes the regression.  Indeed there is some redundancy in the command line options for this regression, but this is not a big deal.